src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -106,13 +106,27 @@ public void remove(ReadOnlyPerson toRemove) throws PersonNotFoundException {
         }
     }
 
```

Comment:
Again, we consider too many empty lines as a coding violation, as it reduces code readability.